packages/features/flags/config.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Right now we only support boolean flags.
+ * Maybe later on we can add string variants or numeric ones
+ **/
+export type AppFlags = {
+  emails: boolean;
+  teams: boolean;
+  webhooks: boolean;
+  workflows: boolean;
+  "booking-page-v2": boolean;
```

Comment:
can we improve this to be v2-booking-page? As mentioned in a example down below v2-workflows